res/values/strings.xml

```diff
@@ -36,18 +36,10 @@
     <string name="menuItem_enableFramelimiter">Sync audio</string>
     <string name="menuItem_disableFramelimiter">Desync audio</string>
     <string name="menuItem_paks">Controller Paks</string> <!-- capitalize since trademark -->
-    <string name="menuItem_pak1_empty">Player 1: Empty</string>
```

Comment:
We could also keep the strings.xml file the same and just parse out the two strings in code. All of the translations stick to the "Player n: Pak Type" format, which is a simple text split on the colon.